default_kms_key_arn.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python2
+import sys
+import json
+import boto3
+
+if __name__ == '__main__':
+    # setup connection to AWS
+    client = boto3.client('kms')
+
+    # get the default KMS key
+    key_aliases = client.list_aliases()['Aliases']
+
+    arn = None
+
+    for alias in key_aliases:
+        if alias['AliasName'] == 'alias/aws/lambda':
+            arn = alias['AliasArn']
+
+    if not arn:
+        json.dump({
+            'default_lambda_kms_key_arn': ""
+        }, sys.stdout, indent=2)
+        sys.stdout.write('\n')
+        sys.exit(1)
```

Comment:
I guess this will stop Terraform from running, but the reason for failure won't be clear. I think it should show an error message along the lines of `could not find default KMS key for Lambda: alias/aws/lambda`.